packages/headless-ui/vaul/test/src/app/controlled/page.tsx

```
@@ -0,0 +1,206 @@
+                    aria-hidden="true"
+                    className="w-3 h-3 ml-1"
+                  >
+                    <path d="M18 13v6a2 2 0 01-2 2H5a2 2 0 01-2-2V8a2 2 0 012-2h6"></path>
+                    <path d="M15 3h6v6"></path>
+                    <path d="M10 14L21 3"></path>
+                  </svg>
+                </a>
+                <a
+                  className="text-xs text-zinc-600 flex items-center gap-0.25"
+                  href="https://twitter.com/emilkowalski_"
+                  target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)